cockroachcloud/cluster-overview-page.md

```diff
@@ -0,0 +1,58 @@
+The **Cluster settings** panel displays the settings you chose during [cluster creation](create-a-free-cluster.html).
+
+| Field     | Description                                                                                             |
+|-----------|---------------------------------------------------------------------------------------------------------|
+| Cloud     | The cluster's [cloud provider](create-a-free-cluster.html#step-2-select-a-cloud-provider-region). |
+| Plan type | The plan type used to create the cluster.                                                               |
+| Region    | The cluster's [region](create-a-free-cluster.html#step-2-select-a-cloud-provider-region).         |
+
+## Usage this month
+
+The **Usage this month** panel displays your cluster usage statistics for the current month.
+
+| Field         | Description                                                                                                                                                                                                                                                                      |
+|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spend limit   | The spend limit you entered when [creating the cluster](create-a-free-cluster.html#step-3-enter-a-spend-limit). Click the **edit** icon to change the spend limit.                                                                                                         |
+| Storage       | The amount of data currently stored in the cluster. This value does not account for compression or replication. The total available storage is determined by your spend limit and Burst RU usage. For details, see [CockroachCloud Architecture](architecture.html#performance). |
+| Request Units | The CPU and I/O resources being used by queries on the cluster, and whether throttling is in effect. The total available RUs are determined by your spend limit and storage usage. For more context, see [CockroachCloud Architecture](architecture.html#concepts).              |
+
+## Cluster statistics panel
+
+This panel displays operational statistics for your cluster.
+
+| Field               | Description                                                                                                                 |
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| Throughput          | The cluster throughput in queries per second (QPS).                                                                         |
+| P99 latency         | The 99th percentile of service latency. Over the last minute, this cluster executed 99 percent of queries within this time. |
```

Comment:
My ear kept catching on this sentence...(so it might just be a me thing) I think it's the "this" in "this cluster executed..." since the rest of the table uses "the cluster". I assume this sentence is sort of an elaboration/more friendly explanation of the P99 latency field. I wondered if: "That is, over the last minute, the cluster executed 99 percent of queries within this time." Or maybe, "For example: over the last minute, the cluster executed..."